demo/config/tools/handbook_creator/default_toc.xsl

```diff
@@ -1,59 +1,59 @@
-<?xml version="1.0" encoding="UTF-8"?>

-<xsl:stylesheet version="1.0"

-                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"

-                xmlns:outline="http://code.google.com/p/wkhtmltopdf/outline"

-                xmlns="http://www.w3.org/1999/xhtml">

-  <xsl:output doctype-public="-//W3C//DTD XHTML 1.0 Strict//EN"

-              doctype-system="http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"

-              indent="yes" />

-  <xsl:template match="outline:outline">

-    <html>

-      <head>

-        <title>Table of Contents</title>

-        <style>

-          h1 {

-            text-align: center;

-            font-size: 20px;

-            font-family: arial;

-          }

-          div {border-bottom: 1px dashed rgb(200,200,200);}

-          span {float: right;}

-          li {list-style: none;}

-          ul {

-            font-size: 20px;

-            font-family: arial;

-          }

-          ul ul {font-size: 80%; }

-          ul {padding-left: 0em;}

-          ul ul {padding-left: 1em;}

-          a {text-decoration:none; color: black;}

-        </style>

-      </head>

-      <body>

-        <h1>Table of Contents</h1>

-        <ul><xsl:apply-templates select="outline:item/outline:item"/></ul>

-      </body>

-    </html>

-  </xsl:template>

-  <xsl:template match="outline:item">

-    <li>

-      <xsl:if test="@title!=''">

-        <div>

-          <a>

-            <xsl:if test="@link">

-              <xsl:attribute name="href"><xsl:value-of select="@link"/></xsl:attribute>

-            </xsl:if>

-            <xsl:if test="@backLink">

-              <xsl:attribute name="name"><xsl:value-of select="@backLink"/></xsl:attribute>

-            </xsl:if>

-            <xsl:value-of select="@title" /> 

-          </a>

-          <span> <xsl:value-of select="@page" /> </span>

-        </div>

-      </xsl:if>

-      <ul>

-        <xsl:apply-templates select="outline:item"/>

-      </ul>

-    </li>

-  </xsl:template>

-</xsl:stylesheet>

+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="2.0"
```

Comment:
This is the only real change besides the newlines. wkhtmltopdf complains about 1.0